contracts/ProjectResolver.sol

```diff
@@ -46,10 +46,12 @@ contract ProjectResolver is SchemaResolver, Initializable, OwnableUpgradeable {
      * This is an bottom up event, called from the attest contract
      */
     function onAttest(
-        Attestation calldata attestation,
+        Attestation calldata /*attestation*/,
         uint256 /*value*/
-    ) internal override returns (bool) {
-        projectOwner[attestation.uid] = attestation.recipient;
+    ) internal pure override returns (bool) {
+        // projectOwner[attestation.uid] = attestation.recipient;
```

Comment:
Thinking more, we should keep this too. Because we want a trail onchain of project ownership and it's transfer. Please revert this too.